com.dropbox.Client.appdata.xml

```diff
@@ -0,0 +1,76 @@
+<description xml:lang="vi"><p>Bạn cần lưu trữ an toàn tất cả các văn bản, hình ảnh, bài hát và các tập tin quan trọng? Dễ dàng thôi! Với ứng dụng này, bạn có thể sử dụng Internet để lưu tất cả mọi thứ quan trọng với bạn. Sau đó, bạn có thể truy cập các tệp này từ bất kỳ máy tính nào có kết nối internet. Sử dụng dịch vụ này để sao lưu các tập tin, chia sẻ hình ảnh hoặc cộng tác trên một dự án bằng cách chia sẻ thư mục Dropbox với bất cứ ai mà bạn muốn. Hãy tạo tài khoản miễn phí và bắt đầu lưu và chia sẻ ngày hôm nay! ** Yêu cầu kết nối Internet</p></description>
+<description xml:lang="zh_CN"><p>想要安全地存储所有重要文件、照片、音乐和文件吗？别急！只要有了这个app，您就可以使用互联网来保存所有重要文件。您还可以从任何有网络连接的计算机上访问这些文件。使用该服务来备份文档，通过共享一个Dorpbox文件夹来共享照片，合作项目。创建一个免费账户，开始保存和分享您的每一天吧！</p></description>
+<description xml:lang="zh_TW"><p>需要安全地儲存你的所有文件、照片、歌曲和檔案嗎？很簡單！使用這款應用程式，你可以使用網路來儲存所有你認為重要的東西。然後你可以從任何一台擁有網路連線的電腦上存取那些檔案。和你想要分享的人一起共享 Dropbox 資料夾，使用這個服務來備份檔案、分享照片或在同一個專案上進行合作。今天就建立免費帳號，並開始儲存和分享吧！ ** 需要網路連線</p></description>
+<description><p>Need to safely store all of your important documents, photos, songs, and files? Easy! With this app you can use the internet to save everything that’s important to you. You can then access those files from any computer that has an internet connection. Use this service to backup your files, share photos or collaborate on a project by sharing a Dropbox folder with whomever you’d like. Create a free account and start saving and sharing today!  ** Requires internet</p></description>
+
+<screenshots><screenshot><image type="source" xml:lang="pt">https://d3lapyynmdp1i9.cloudfront.net/screenshots/com.dropbox.Client/pt/com.dropbox.client-screenshot1.jpg</image>
+<image type="source">https://d3lapyynmdp1i9.cloudfront.net/screenshots/com.dropbox.Client/C/com.dropbox.client-screenshot1.jpg</image>
+<image type="source" xml:lang="es">https://d3lapyynmdp1i9.cloudfront.net/screenshots/com.dropbox.Client/es/com.dropbox.client-screenshot1.jpg</image>
+</screenshot>
+<screenshot><image type="source" xml:lang="pt">https://d3lapyynmdp1i9.cloudfront.net/screenshots/com.dropbox.Client/pt/com.dropbox.client-screenshot2.jpg</image>
+<image type="source">https://d3lapyynmdp1i9.cloudfront.net/screenshots/com.dropbox.Client/C/com.dropbox.client-screenshot2.jpg</image>
+<image type="source" xml:lang="es">https://d3lapyynmdp1i9.cloudfront.net/screenshots/com.dropbox.Client/es/com.dropbox.client-screenshot2.jpg</image>
+</screenshot>
+</screenshots>
+
+<metadata>
+  <value key="EndlessOS::legacy-ext-runtime-installed">d8aee708de68406b036f323f0ad06f0f4353c133328d471b84d2ff9a52ae30da</value>
+  <value key="EndlessOS::available-since">3.1</value>
```

Comment:
I think these two can be removed safely now. They are to do with the move from the old "endless extra data" apps in 3.0 to the upstream solution in 3.1.

Reply:
Oops! I removed those from the json file but forgot these ones. Will remove them